docs/libraries/v4-utils-js/index.md

```diff
@@ -0,0 +1,152 @@
+# Utility Library
+![Tests](https://github.com/pooltogether/v4-utils-js/actions/workflows/main.yml/badge.svg)
+[![Coverage Status](https://coveralls.io/repos/github/pooltogether/v4-utils-js/badge.svg?branch=main)](https://coveralls.io/github/pooltogether/v4-utils-js?branch=main)
+![ts](https://badgen.net/badge/-/TypeScript?icon=typescript&label&labelColor=blue&color=555555)
+[![GPLv3 license](https://img.shields.io/badge/License-GPLv3-blue.svg)](http://perso.crans.org/besson/LICENSE.html)
+![npm](https://img.shields.io/npm/v/@pooltogether/v4-utils-js)
+
+The `@pooltogether/v4-utils-js` [node module package](https://www.npmjs.com/package/@pooltogether/v4-utils-js) provides calculations, computations and general utility functions for the PoolTogether V4 protocol.
+
+As normally the case, the utility library was designed to be modular. Enabling developers to more easily use low-level primitives to create the higher-level computations/operations. High-order operations like `winningPicks()` which takes an `address`, plus `Draw` and `PrizeDistribution` EVM structs fetched via `v4-js-client`: returning encoded transactions **ready for any EVM network**.
+
+Helping with both **low-level calculations** and **higher-order computations**, plus **essential utilities** like encoding, filtering and sorting.
+
+**Join the PoolTogether Discord, ask questions and get help from the community.**
+
+[![Discord](https://badgen.net/badge/icon/discord?icon=discord&label)](https://discord.gg/JFBPMxv5tr)
```

Comment:
We have discord invites turned off so you need to send the users to pooltogether.com/discord